tests/system/feature_store/test_feature_store.py

```diff
@@ -396,6 +396,7 @@ def test_multiple_entities(self):
             operations=["sum", "max"],
             windows=["1h"],
             period="10m",
+            emit_policy=EmitAfterMaxEvent(1),
```

Comment:
I don't fully understand what this emit policy is doing, but I'm pretty sure adding this line only makes the new logic to run, while we also want to test that the new logic actually works - i.e. I assume adding the emit policy affects the results somehow - let's assert that the results are really different when we use and don't use the emit policy.
Also, generally, I prefer to have small contained tests to test a specific feature, instead of one big test which tries to cover all

Reply:
Emit policy is thoroughly tested in storey. This test is only to make sure that the parameter is passed to storey (do not take this part for granted :)))